src/components/user-dropdown-disclosure/components/debugging-items.tsx

```diff
@@ -0,0 +1,48 @@
+import { IconAlertCircle16 } from '@hashicorp/flight-icons/svg-react/alert-circle-16'
+import { useDebuggingState } from 'contexts'
+import useAuthentication, {
+	DEBUGGING_ERROR_PROPERTY_NAME,
+} from 'hooks/use-authentication'
+import {
+	DropdownDisclosureButtonItem,
+	DropdownDisclosureLabelItem,
+	DropdownDisclosureSeparatorItem,
+} from 'components/dropdown-disclosure'
+
+/**
+ * A special portion of the `UserDropdownDisclosure` that is intended to only
+ * render if the `showDebugUserMenuItems` feature flag is enabled. The purpose
+ * is to add custom items that can be used to trigger the app to change state.
+ *
+ * For example, an authentication error is tedious and challenging
+ * to organically reproduce. With this component, we can render an item that
+ * emulates an error so we can test how the app handles the error.
+ */
```

Comment:
future 💭  these dev-only commands feel like something that would be well suited to integrating into the command dialog.

Reply:
Ooooh I like that idea too! Reminds me of being able to emulate `prefers-reduced-motion` in Chrome dev tools with a command palette.

Reply:
🗒️ PR has been updated and this convo is no longer relevant, so marking resolved.